modules/3_fips/fips.tf

```diff
@@ -87,8 +91,8 @@ resource "null_resource" "bastion_fips_enable_and_reboot" {
       <<EOF
 # enable FIPS as required
 sudo fips-mode-setup --enable
-shutdown -r now
+sudo shutdown -r +1
 EOF
```

Comment:
Great work figuring this out.

The `+1` is a minute or 1 second? if 1 second, let's increase to 5 seconds (just in case)

Reply:
@prb112 `+1` is a minute in this case.